### DISCUSSION TITLE
Weston RDP example

### DISCUSSION THREAD

**Discussion by leowang1225** [microsoft/wslg#177] [Q&A]
I have build Weston, and run “weston --backend=rdp-backend.so”. Is there any example to test rdp?

**Comment by spronovo** [comment]
Hello, which version of Weston are you building? From the official Wayland project (https://github.com/wayland-project/weston) or the mirror we are using for WSLg (https://github.com/microsoft/weston-mirror). Are you trying to run Weston in WSL2? Or in a native Linux system?

When you run Weston against the RDP backend as above, it will open a tcp port to listen on (port 3389). From another PC, you need to be running an RDP client and connect to that system IP (i.e. on Windows you can use mstsc.exe).

**Reply by leowang1225** [reply]
The version of Weston is WSLg, I try to run a native Linux System first, then I will try in WSL2.

When you run Weston against the RDP backend as above, it will open a tcp port to listen on (port 3389).
--> I want to know whether where is the sample code to test RDP backend works  in a native linux system
